docs/source/guides/area_detectors.rst

```diff
@@ -0,0 +1,721 @@
+
+Complete Area Detector Guide
+-----------------------------
+
+Understanding Area Detector Architecture
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Area detectors in BITS follow the EPICS Area Detector architecture:
+
+.. code-block:: text
+
+    Area Detector Components:
+    ├── Camera (cam)          # Detector control
+    ├── Plugins/              # Image processing
+    │   ├── image             # Live image display
+    │   ├── stats             # Statistics calculation
+    │   ├── roi               # Region of interest
+    │   ├── transform         # Image transforms
+    │   └── hdf5              # File saving
+    └── Configuration         # BITS integration
+
+**BITS provides three approaches:**
+
+1. **apstools Factory** (Recommended) - Automatic plugin setup
+2. **Version-Compatible Classes** - Handle EPICS version differences
+3. **Custom Detector Classes** - Full customization
+
+Using apstools Area Detector Factory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Automatic Detector Creation:**
+
+.. code-block:: python
+
+    # devices/detectors.py - Factory approach
+    from apstools.devices import ad_creator
+
+    # Create simulation detector with standard plugins
+    adsim = ad_creator(
+        "IOC:ADSIM:",
+        name="adsim",
+        detector_class="SimDetectorCam",
+        plugins=["image", "stats1", "roi", "hdf1"]  # Note: Use numbered convention (hdf1, stats1)
+    )
+
+    # Advanced factory configuration
+    advanced_detector = ad_creator(
+        "IOC:DETECTOR:",
+        name="advanced_det",
+        detector_class="SimDetectorCam",  # Use SimDetectorCam for development
+        plugins=["image", "stats1", "roi", "hdf1"]  # Note: Use numbered convention
+    )
+
+.. note::
+   For production detectors, replace ``SimDetectorCam`` with actual detector
+   camera classes like ``PilatusDetectorCam``, ``PerkinElmerDetectorCam``, etc.
+
+**Factory Benefits:**
+- **Automatic plugin configuration**: No need to manually set up plugin chains
+- **Proper port connections**: Data flows correctly between camera and plugins  
+- **Standard naming conventions**: Uses established patterns (stats1, hdf1, etc.)
+- **Built-in error handling**: Factory validates configuration before creation
+- **Immediate functionality**: Creates working detectors that can acquire data
+
+.. note::
+   The numbered plugin convention (hdf1, stats1, etc.) allows for multiple
+   plugins of the same type. For example, you could have hdf1 for raw data
+   and hdf2 for processed data, or stats1 from camera and stats2 from ROI.
+
+Version Compatibility Patterns
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Handling EPICS Area Detector Version Changes:**
+
+.. code-block:: python
+
+    # devices/area_detector.py - Version compatibility pattern
+    from apstools.devices import CamMixin_V34
+    from ophyd.areadetector import CamBase
+    from ophyd.areadetector.cam import SimDetectorCam
+
+    class CamUpdates_V34(CamMixin_V34, CamBase):
```

Comment:
This class is not necessary.  It repeats code already existing in `CamMixin_V34`.  Please remove and refactor below.